TombLib/TombLib/Catalogs/TEN Node Catalogs/Timers.lua

```diff
@@ -0,0 +1,274 @@
+local Timer = require("Engine.Timer")
+LevelFuncs.noA = function() end
+
+-- !Name "Create Timer with Function"
+-- !Conditional "False"
+-- !Description "after a specified number of seconds, the specified thing happens"
+-- !Section "Timer"
+-- !Arguments "NewLine, String, 57,[ NoMultiline ], Timer name"
+-- !Arguments "Numerical, 30, [ 0 | 1000 | 2 ], The duration of the timer in seconds"
+-- !Arguments "Boolean , 13, Loop"
+-- !Arguments "NewLine, Boolean , 33, Show Minutes" "Boolean , 33, Show Seconds" "Boolean , 33, Show Deciseconds"
+-- !Arguments "NewLine, LuaScript, The function to call when the time is up"
+LevelFuncs.Engine.Node.CreateTimerWithFunction = function(name, time, loop, minutes, seconds, deciseconds, fucs)
```

Comment:
`fucs`?

Reply:
I renamed the **_fucs_** parameter to **_LuaFunction_**